torch_geometric/explain/explanation.py

```diff
@@ -245,6 +245,53 @@ def visualize_feature_importance(
 
         return ax
 
+    def visualize_subgraph(self, node_index: int) -> str:
+        r"""Threshold the explanation mask according to the thresholding method.
+        Generate and explanation subgraph visualization for thresholded and non-thresholded explanation.
+        The output would be a visualization of the explanation subgraph where edge width depends on the importance value of each edge.
+        Args:
+            explanation (Explanation): The explanation to visualize.
+        """
+        import graphviz
+
+        u_nodes = []
+        v_nodes = []
+        edge_weights = []
+
+        for index, weight in enumerate(self.edge_mask):
+            # edge_index is in COO format
+            u_node = int(self.edge_index[0][index].detach().item())
+            v_node = int(self.edge_index[1][index].detach().item())
+            if (weight != 0):
+                u_nodes.append(u_node)
+                v_nodes.append(v_node)
+                edge_weights.append(weight.detach().item())
+
+        # normalize edge weights between 0 and 10 to use them as edge thickness in visualized graph
+        norm_edge_weights = []
+
+        for weight in edge_weights:
+            norm = (float(weight) - min(edge_weights)) / (max(edge_weights) -
+                                                          min(edge_weights))
+            norm_edge_weights.append(round(norm * 10, 2))
+
+        # Initialize a graphviz undirected graph
+        graph_file = "visualized_subgraph"
+        file_format = "pdf"
```

Comment:
Lets make these user arguments. The values here can be default but the user can change it if needed.